website/static/js/home-page/quickProjectSearchPlugin.js

```diff
@@ -46,7 +47,7 @@ var QuickSearchProject = {
         self.templateNodes.start();
 
         // Load up to first ten nodes
-        var url = $osf.apiV2Url('users/me/nodes/', { query : { 'embed': 'contributors'}});
+        var url = $osf.apiV2Url('users/me/nodes/', { query : { 'embed': ['contributors','root', 'parent']}});
```

Comment:
I'm not opposed to checking, but it's, what, 10 records at most? If we can't do that easily, we have bigger troubles.